socorro/processor/rules/mozilla.py

```diff
@@ -1152,6 +1152,12 @@ def action(self, raw_crash, dumps, processed_crash, processor_meta_data):
         processor_meta_data["processor_notes"].extend(result.notes)
         if "proto_signature" in result.extra:
             processed_crash["proto_signature"] = result.extra["proto_signature"]
+        if "crashing_thread_id" in result.extra:
+            processed_crash["crashing_thread_id"] = result.extra["crashing_thread_id"]
+        if "crashing_thread_name" in result.extra:
+            processed_crash["crashing_thread_name"] = result.extra[
+                "crashing_thread_name"
+            ]
```

Comment:
These aren't related to signature generation, so they shouldn't be here.

Extracting the `crashing_thread_id` and `crashing_thread_name` fields from the stackwalker output and adding them to the processed crash should happen in the `CrashingThreadInfoRule`:

https://github.com/mozilla-services/socorro/blob/6e7aadc03a2ba5315a978fac433285dde8758af3/socorro/processor/rules/breakpad.py#L24-L53